storage/badger/approvals.go

```diff
@@ -37,11 +37,10 @@ func NewResultApprovals(collector module.CacheMetrics, db *badger.DB) *ResultApp
 
 	res := &ResultApprovals{
 		db: db,
-		cache: newCache(collector,
+		cache: newCache(collector, metrics.ResourceResult,
```

Comment:
I would suggest to change it into ResourceApproval.

Reply:
Yep, good idea. Changed